src/compiler.ts

```diff
@@ -3570,7 +3574,9 @@ export class Compiler extends DiagnosticEmitter {
       }
       this.error(
         DiagnosticCode.Type_0_is_not_assignable_to_type_1,
-        reportNode.range, fromType.toString(), toType.toString()
+        reportNode.range,
+        fromType.toString(),
+        toType.toString()
```

Comment:
So far I used the convention (for readability of the overall file) to not expand this too much, as it doesn't really add much. I somewhat preferred the old way, or are there good reasons to make this more verbose?